### DISCUSSION TITLE
现在无法进行离线激活halo已经插件

### DISCUSSION THREAD

**Discussion by dlluckboy** [halo-dev/halo#7168] [Q&A]
### 前置条件

- [x] 已经在 [Issues](https://github.com/halo-dev/halo/issues) 列表中搜索了相关问题。
- [x] 这是和 Halo 相关的特性，如果是非项目本身的问题，建议在 [Discussions](https://github.com/halo-dev/halo/discussions) 提交。
- [x] 如果是插件和主题特性建议，请在对应的插件和主题仓库提交。

### 你当前使用的版本

2.20.12

### 描述一下此特性

halo无法离线进行激活，这个太不方便了，企业内部使用，好多情况都是内网环境下使用，系统能增加离线激活方式。

**Comment by ruibaby** [comment]
这个问题已经在凌霞内部记录，后续将在凌霞支持下载付费应用和离线许可证。
